docs/src/tutorials/tutorial_1.md

```diff
@@ -4,4 +4,336 @@
+* `actions`
+* `pathname`
+
+### pathname
+
+The `pathname` is a String. In order to get the final result of `javis`, you will need to set `pathname` to the path where the gif will be stored and the gif name. Similar to the following:
+
+```julia
+javis(
+    [[video]], # Video struct
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Video
+
+The `video` is a Video struct and its constructor arguments are width and height.
+
+```julia
+video = Video(500,500) # 500x500 // width x height
+javis(
+    video,
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Actions
+
+The `actions` is a Vector of actions. There are two types of actions:
+
+* `BackgroundAction`
+* `Action`
+
+```julia
+video = Video(500,500)
+javis(
+    video,
+    [
+        ...
+    ],
+    pathname="/path/animation.gif"
+)
+```
+
+#### BackgroundAction()
+
+```julia
+BackgroundAction(
+    [[frames]], # Range (Int:Int)
+    [[func]] # Function call
+)
+```
+
+* `frames`: a range of frames it is applied to.
+* `func`: the function that is called for each frame (in this example, we will be using `ground()`).
+
+##### ground()
+
+```julia
+function ground(args...) 
+    background("white") # canvas background
+    sethue("black") # pen color
+end
+```
+
+The `ground` function sets the background to white and the paint brush color to black.
+
+**Why do I need the `args...`?** Each user function gets three arguments `video, action, frame` in order:
+* `Video`: Video struct
+* `Action`: Action struct
+* `frame`: the current frame number
+
+They are irrelevant for the background such that we don't need to write them down explicitly. "Unfortunately" we need to write `args...` such that Julia actually knows that we have a method that accepts those three arguments. The `...` basically stands for as many arguments as you want.
+
+Note: `sethue` is the same as `setcolor` but doesn't mess with the opacity
+
+
+```julia
+BackgroundAction(
+    1:70, # 70 frames
+    ground # apply ground function for each frame
+)
+```
+
+![](assets/background.gif)
+
+#### Action()
+
+```julia
+Action(
+    [[frames]], # Range (Int:Int)
+    :[[variable_name]], # Variable name
+    (args...)->
+        [[func]] # Function call
+    , 
+    [[transition]] # Transition struct (optional)
+)
+```
+* `frames`: Range of frames it is applied to.
+* `variable_name`: Variable name for the action to be applied on.
+* `func`: Function call to set action
+* `transition` (optional): Transition struct (for this tutorial, we will be using `Rotation()`)
+
+```julia
+Action(
+    1:70, # 70 frames
+    :red_ball, # variable name
+    (args...)->
+        [[func]] # function call
+    , 
+    [[transition]] # Transition struct (optional)
+)
+```
+
+[`sethue`](), [`circle`](), and [`line`]() are path of `Luxon.jl`, so we are using them to define `object()`, `track()`, and `connector()`. These functions will be used in this tutorials to replace `func` in `Action()`.
```

Comment:
the links are missing here. You can link to https://juliagraphics.github.io/Luxor.jl/stable/colors-styles/#Luxor.sethue as an example.

A small typo it's Luxor not Luxon :wink:

Reply:
Oh and you still use `path!` later and not `track`